wandb/sdk/data_types/image.py

```diff
@@ -297,10 +297,21 @@ def _initialize_from_data(
             "PIL.Image",
             required='wandb.Image needs the PIL package. To get it, run "pip install pillow".',
         )
+
+        accepted_formats = ["png", "jpg", "jpeg", "bmp"]
+        if file_type is None:
+            self.format = "png"
+        else:
+            self.format = file_type
```

Comment:
```suggestion
        self.format = file_type or "png" 
```
should work for None and any false-y values